Content.Server/Nutrition/Components/FoodComponent.cs

```diff
@@ -6,87 +6,94 @@
-
-        /// <summary>
-        /// Acceptable utensil to use
-        /// </summary>
-        [DataField("utensil")]
-        public UtensilType Utensil = UtensilType.Fork; //There are more "solid" than "liquid" food
-
-        /// <summary>
-        /// Is utensil required to eat this food
-        /// </summary>
-        [DataField("utensilRequired")]
-        public bool UtensilRequired = false;
-
-        /// <summary>
-        ///     If this is set to true, food can only be eaten if you have a stomach with a
-        ///     <see cref="StomachComponent.SpecialDigestible"/> that includes this entity in its whitelist,
-        ///     rather than just being digestible by anything that can eat food.
-        ///     Whitelist the food component to allow eating of normal food.
-        /// </summary>
-        [DataField("requiresSpecialDigestion")]
-        public bool RequiresSpecialDigestion = false;
-
-        /// <summary>
-        ///     Stomachs required to digest this entity.
-        ///     Used to simulate 'ruminant' digestive systems (which can digest grass)
-        /// </summary>
-        [DataField("requiredStomachs")]
-        public int RequiredStomachs = 1;
-
-        /// <summary>
-        /// The localization identifier for the eat message. Needs a "food" entity argument passed to it.
-        /// </summary>
-        [DataField("eatMessage")]
-        public string EatMessage = "food-nom";
-
-        /// <summary>
-        /// How long it takes to eat the food personally.
-        /// </summary>
-        [DataField("delay")]
-        public float Delay = 1;
-
-        /// <summary>
-        ///     This is how many seconds it takes to force feed someone this food.
-        ///     Should probably be smaller for small items like pills.
-        /// </summary>
-        [DataField("forceFeedDelay")]
-        public float ForceFeedDelay = 3;
-
-        [ViewVariables]
-        public int UsesRemaining
+        get
         {
-            get
+            if (!EntitySystem.Get<SolutionContainerSystem>().TryGetSolution(Owner, SolutionName, out var solution))
             {
-                if (!EntitySystem.Get<SolutionContainerSystem>().TryGetSolution(Owner, SolutionName, out var solution))
-                {
-                    return 0;
-                }
+                return 0;
+            }
 
-                if (TransferAmount == null)
-                    return solution.Volume == 0 ? 0 : 1;
+            if (TransferAmount == null)
+                return solution.Volume == 0 ? 0 : 1;
 
-                return solution.Volume == 0
-                    ? 0
-                    : Math.Max(1, (int) Math.Ceiling((solution.Volume / (FixedPoint2)TransferAmount).Float()));
-            }
+            return solution.Volume == 0
+                ? 0
+                : Math.Max(1, (int) Math.Ceiling((solution.Volume / (FixedPoint2)TransferAmount).Float()));
         }
```

Comment:
i will give you a cookie if you move this sin against mankind to the system

Reply:
send cookie to 308 negra arroyo lane, albequerque, new space mexico